docs/platform/howto/byoc/assign-project-custom-cloud.md

```diff
@@ -1,111 +1,108 @@
 ---
-title: Enable your AWS custom cloud in Aiven organizations, units, or projects
+title: Enable a custom cloud in Aiven organizations, units, or projects
 sidebar_label: Attach projects
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 import ConsoleLabel from "@site/src/components/ConsoleIcons";
 
-Configure the availability of your [custom cloud](/docs/platform/concepts/byoc) to access
-it in all the projects, selected organizational units, or specific projects only.
+Select your organizations, units, or project that can access and use your [custom cloud](/docs/platform/concepts/byoc).
 
 ## About making custom clouds available from your projects
```

Comment:
Maybe change "About making" to "How to make"

Reply:
Actually, this section is only to give some context for the upcoming operation the user will to introduced to in the next sections. Saying "How to" might set some expectations for steps required for enabling projects on custom clouds, which is not the purpose of this "About.." section